spec/support/schemas/rating_info_response.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {},
+  "properties": {
+    "data": {
+      "properties": {
+        "attributes": {
+          "properties": {
+            "service_connected_combined_degree": {
+              "type": "string"
+            },
+            "user_percent_of_disability": {
+              "type": "string"
```

Comment:
should this match the type in the object?

Reply:
Yes...thank you...good catch.  I updated the other files but forgot this one.